setup.py

```diff
@@ -18,8 +18,7 @@
 AUTHOR_EMAIL = 'xi@resolvent.net'
 LICENSE = "MIT"
 PLATFORMS = "Any"
-URL = "http://pyyaml.org/wiki/PyYAML"
-DOWNLOAD_URL = "http://pyyaml.org/download/pyyaml/%s-%s.tar.gz" % (NAME, VERSION)
```

Comment:
DOWNLOAD_URL is used below. You need to either remove all references to it, or change the link.

Reply:
Thanks, I've put DOWNLOAD_URL back. I see it still contains valid files for the latest release.